src/service/repo/anonymousMessageRepo.ts

```diff
@@ -80,28 +81,44 @@ async function sendAnonymousMessage(
 }
 
 async function getAllAnonymousChannels() {
+  // get time stamp
+  const timeStamp = StorageUtils.channelAnonTimeStamps.get();
+  const url = timeStamp
+    ? `${baseUrl.getAllAnonymousChannels}?last_fetch_date=${timeStamp}`
+    : baseUrl.getAllAnonymousChannels;
   try {
-    const response = await anonymousApi.get(baseUrl.getAllAnonymousChannels);
+    const response = await anonymousApi.get(url);
     if (response.status === 200) {
+      console.log('SUCCESS URL', url);
       return Promise.resolve(response.data?.data);
     }
 
     return Promise.reject(response.data?.status);
   } catch (e) {
+    console.log('FAILED URL', url);
+
     console.log(e);
     return Promise.reject(e);
   }
 }
 
 async function getAllAnonymousPostNotifications() {
+  const timeStamp = StorageUtils.anonymousNotificationTimeStamp.get();
+  const url = timeStamp
+    ? `${baseUrl.getAllAnonymousPostNotifications}?last_fetch_date=${timeStamp}`
+    : baseUrl.getAllAnonymousPostNotifications;
+
   try {
-    const response = await anonymousApi.get(baseUrl.getAllAnonymousPostNotifications);
+    const response = await anonymousApi.get(url);
     if (response.status === 200) {
+      console.log('SUCCESS URL', url);
       return Promise.resolve(response.data?.data);
     }
 
     return Promise.reject(response.data?.status);
   } catch (e) {
+    console.log('FAILED URL', url);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous hunk, these changes introduce timestamp-based fetching for anonymous post notifications. The same comments apply here: this is a good performance improvement, but error handling for potential failures of `StorageUtils.anonymousNotificationTimeStamp.get()` should be added.
```diff
+  async function getAllAnonymousPostNotifications() {
+   const timeStamp = StorageUtils.anonymousNotificationTimeStamp.get();
+   const url = timeStamp
+     ? `${baseUrl.getAllAnonymousPostNotifications}?last_fetch_date=${timeStamp}`
+     : baseUrl.getAllAnonymousPostNotifications;
+
+    try {
+     const response = await anonymousApi.get(url);
+      if (response.status === 200) {
+       console.log('SUCCESS URL', url);
+        return Promise.resolve(response.data?.data);
+      }
+  
+      return Promise.reject(response.data?.status);
+    } catch (e) {
+     console.log('FAILED URL', url);
+
+     console.log(e);
+     return Promise.reject(e);
+   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->